src/open_inwoner/components/templates/components/Notification/Notifications.html

```diff
@@ -1,7 +1,13 @@
-{% load notification_tags %}
+{% load notification_tags string_tags %}
 
 <div class="notifications">
     {% for message in messages %}
-        {% notification icon=message.icon type=message.level_tag message=message.message closable=True compact=compact %}
+        {% with as_markdown=message.extra_tags|is_substring:"as_markdown" %}
```

Comment:
`extra_tags|has_as_substring:"as_markdown"` would read more natural imho (similarly for the signature of the tag).